crowdin.yml

```diff
@@ -20,30 +24,60 @@ files:
   # dest: crowdin path in https://crowdin.com/project/keyman
   # translation: local path where downloaded translations from crowdin go
 
+
   # Android files
-  - source: /KMEA/app/src/main/res/values/strings.xml
+
+  - source: /android/KMEA/app/src/main/res/values/strings.xml
     dest: /android/KMEA/strings.xml
-    translation: /KMEA/app/src/main/res/values-%android_code%/strings.xml
+    translation: /android/KMEA/app/src/main/res/values-%android_code%/strings.xml
     languages_mapping:
       # Prevent invalid region pap-rPAP
       android_code:
         pap: pap
 
-  - source: /KMAPro/kMAPro/src/main/res/values/strings.xml
+  - source: /android/KMAPro/kMAPro/src/main/res/values/strings.xml
     dest: /android/KMAPro/strings.xml
-    translation: /KMAPro/kMAPro/src/main/res/values-%android_code%/strings.xml
+    translation: /android/KMAPro/kMAPro/src/main/res/values-%android_code%/strings.xml
     languages_mapping:
       # Prevent invalid region pap-rPAP
       android_code:
         pap: pap
 
+  # Windows files
+
+  - source: /windows/src/desktop/locale/strings.xml
+    dest: /windows/strings.xml
+    translation: /windows/src/desktop/locale/%locale%/strings.xml
+    type: android
```

Comment:
Is this the right type for windows?

Reply:
Yes -- we'll be using the Android format because it's close to the existing locale.xml format we use on Windows. I plan to convert the Windows code to use strings.xml format (locale.xml is my own invention anyway)